docs/content/docs/test/Run.md

```diff
@@ -1,39 +1,111 @@
 ---
-title: Введение
+title: Запуск тестов
 type: docs
 ---
 
 # Запуск тестов
 
-## IDE
+{{<avito page>}}
+
+В данном документе собраны способы запуска unit и instrumentation тестов. Описаны наборы тестов, которые запускаются
```

Comment:
This is extra information. We will have a page structure in a generated page. We can delete these sentences.
IMO

Reply:
Removed